src/components/FileUpload/FileUpload.tsx

```diff
@@ -111,7 +111,8 @@ export const FileUpload: React.FC<FileUploadProps> = ({
 	const { getRootProps, getInputProps, isDragActive, open } = useDropzone({
 		accept: acceptedFilePattern ?? {
 			'image/*': ['.png', '.jpg', '.jpeg', '.gif'],
-			'application/pdf': ['.pdf']
+			'application/pdf': ['.pdf'],
+			'text/csv': ['.csv']
```

Comment:
Please don't add this as a default behaviour. Possibility to upload CSV file is not common and hasn't been used in any of the projects so far. Not everywhere we have `acceptedFilePattern` defined because the default accept worked fine. And if we added it here, it would allow uploading CSV files on those places.

Reply:
Ok, makes sense. I removed it.